### ISSUE TITLE
Accessibility issues with Tree

### ISSUE THREAD

**Issue by haojia321** [SAP/ui5-webcomponents#1892]
**Issue**
While using screen reader to access Tree component, we noticed this a11y issue. "Expanded" or "Collapsed" is read by screen reader.

**Expected behavior**
"Expanded" and "Collapsed" should not be announced by screen reader if it is a leaf node.

We used NVDA as screen reader.

**Comment by mayank2102** [comment]
I am facing issue that a dragon plugin which acts as a screen reader is unable to read the expand/colapse button as the element doesn't have a name...do anyone of you have a solution for this?
